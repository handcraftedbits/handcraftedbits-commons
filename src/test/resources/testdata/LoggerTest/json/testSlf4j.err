{"@version":"1","message":"warn from Slf4j","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"WARN","level_value":30000}
{"@version":"1","message":"warn from Slf4j with MDC","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"WARN","level_value":30000,"key1":"value1","key2":"value2"}
{"@version":"1","message":"error from Slf4j","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"ERROR","level_value":40000}
{"@version":"1","message":"error from Slf4j with MDC","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"ERROR","level_value":40000,"key1":"value1","key2":"value2"}
