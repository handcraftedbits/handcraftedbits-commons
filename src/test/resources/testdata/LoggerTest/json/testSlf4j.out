{"@version":"1","message":"trace from Slf4j","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"TRACE","level_value":5000}
{"@version":"1","message":"trace from Slf4j with MDC","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"TRACE","level_value":5000,"key1":"value1","key2":"value2"}
{"@version":"1","message":"debug from Slf4j","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"DEBUG","level_value":10000}
{"@version":"1","message":"debug from Slf4j with MDC","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"DEBUG","level_value":10000,"key1":"value1","key2":"value2"}
{"@version":"1","message":"info from Slf4j","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"INFO","level_value":20000}
{"@version":"1","message":"info from Slf4j with MDC","logger_name":"com.handcraftedbits.commons.api.log.LoggerTest","thread_name":"main","level":"INFO","level_value":20000,"key1":"value1","key2":"value2"}
